src/hostname/hostnamectl.c

```diff
@@ -107,6 +109,13 @@ static void print_status_info(StatusInfo *i) {
         if (!isempty(i->architecture))
                 printf("      Architecture: %s\n", i->architecture);
 
+        if (!isempty(i->vendor))
+                printf("            Vendor: %s\n", i->vendor);
+
+        if (!isempty(i->model))
+                printf("             Model: %s\n", i->model);
+
+
```

Comment:
spurious empty lines

Reply:
The "Architecture" line had 6 spaces prefixing and yours has 4.  (Not sure if that matters though)

Reply:
my understanding is that we want to align the semicolons one below the other on the output of "hostnamectl", isn't it?

Reply:
(I'm not sure the style here - @poettering should confirm it, I just noted the difference when glancing through the PR)

Reply:
yes, this is aligned along the colons